### PR TITLE
Updated Upstream (BungeeCord)

### PR DESCRIPTION
**Pull request by NoahvdAa**
Upstream has released updates that appear to apply and compile correctly.
This update has not been tested by PaperMC and as with ANY update, please do your own testing

BungeeCord Changes:
eccdf87f Minecraft 1.19 support

### CODE REVIEW

**Comment by NoahvdAa** [comment]
It builds now! Haven't done any testing yet though...

File: BungeeCord-Patches/0035-Use-Log4j2-for-logging-and-TerminalConsoleAppender-f.patch [hunk 1]
```diff
@@ -232,8 +232,311 @@ index 00000000..cfd039cd
...
++io/github/waterfallmc/waterfall/log4j/Log4JLogHandler.class
+diff --git a/log4j/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst b/log4j/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst
+new file mode 100644
+index 00000000..ba7fac50
+--- /dev/null
++++ b/log4j/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst
+@@ -0,0 +1,2 @@
++/Users/noah/Documents/Waterfall/Waterfall-Proxy/log4j/src/main/java/io/github/waterfallmc/waterfall/log4j/WaterfallLogger.java
++/Users/noah/Documents/Waterfall/Waterfall-Proxy/log4j/src/main/java/io/github/waterfallmc/waterfall/log4j/Log4JLogHandler.java
+diff --git a/log4j/target/waterfall-log4j-1.18-R0.1-SNAPSHOT.jar b/log4j/target/waterfall-log4j-1.18-R0.1-SNAPSHOT.jar
```
Comment (electronicboy) [inline BungeeCord-Patches/0035-Use-Log4j2-for-logging-and-TerminalConsoleAppender-f.patch]: bad

**Comment by NoahvdAa** [comment]
Sorry for the messy commits, should be fixed now

**Review by electronicboy — APPROVED** [review]

**Comment by KaiKikuchi** [comment]
It compiles, but running it instantly gives an error. I tried with Java 8 and Java 17, same error. This is the entire output.

```
> java -jar Waterfall.jar
Exception in thread "main" java.util.MissingResourceException: Can't find bundle for base name messages, locale en
        at java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:1581)
        at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1396)
        at java.util.ResourceBundle.getBundle(ResourceBundle.java:854)
        at net.md_5.bungee.BungeeCord.<init>(BungeeCord.java:198)
        at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:64)
        at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15)

```

**Comment by MiniDigger** [comment]
thanks!

**Pull request merged**